src/Accelerators/NNPA/Dialect/ZHigh/ZHighHelper.cpp

```diff
@@ -154,5 +155,21 @@ ZTensorEncodingAttr::DataLayout getZTensorLayout(Type type) {
   return ZTensorEncodingAttr::DataLayout::UNDEFINED;
 }
 
+Value getMinusBcastConst(
+    mlir::OpBuilder &builder, Location loc, FloatAttr floatAttr, Value X) {
+  ShapedType xType = X.getType().cast<ShapedType>();
+  assert(xType.hasStaticShape() && "expected static shape");
+  Type resType =
+      RankedTensorType::get(xType.getShape(), xType.getElementType());
```

Comment:
Why not just using `X.getType()` that is the same type instead of creating a new `resType`?

Reply:
Fixed. Thank you.